pages/dkp/kaptain/1.3.0/install/air-gapped-dkp/index.md

```diff
@@ -64,6 +65,7 @@ Use the following instructions to install Spark Operator from Kommander Catalog
     IFS=$'\n\t'
 
     readonly AIRGAPPED_TAR_FILE=${AIRGAPPED_TAR_FILE:-"kaptain-image-bundle.tar"}
+    <!-- NEW TAR because of Minio in this whole code block?-->
```

Comment:
Is this needed now?

Reply:
Yes. This is where we need them to extract the NOTICES.txt and the image bundle which will also be a tar.
```suggestion
  Extract the NOTICES.txt and the image bundle archive before use.
  
  ```bash
  tar -xvf kaptain-air-gapped-1.3.0.tar.gz
  ```
    <!-- NEW TAR because of Minio in this whole code block?-->
```

Reply:
Since this is a code block/script, how do we best approach the steps? Should we tell customers to extract first and then just keep the code block as it was? Or do we need to change something in the code block itself?

Reply:
Actually not here... At the previous location. This script block doesn't change. Good catch @gvasquezvargas!